code/game/machinery/kitchen/processor.dm

```diff
@@ -116,6 +116,10 @@
 		feedback_add_details("slime_core_harvested","[replacetext(S.colour," ","_")]")
 	..()
 
+/datum/food_processor_process/mob/voxchicken //Do not relocate please I will cry; I tried sorting all fancylike with some help, but it seems the file is scuffed beyond my mortal means
```

Comment:
For clarity to other readers: We found that the current way the code works is so idiotic, there's no reasonable way to implement a priority system in this case without significant changes. They wouldn't be that hard but IMO they're significant enough to be out of scope for this PR.